### ISSUE TITLE
comment on transmembrane transport

### ISSUE THREAD

**Issue by gocentral** [geneontology/go-ontology#11180]
In the discussion of #10968 MP uracil import into cell, David O-S raised the issue of entities taken up by some form of endocytosis. Those entities certainly change from an extracellular location to an intracellular one without crossing a membrane, but I can't think of a case\* where an intravesicular entity gets into the cytosol without crossing a membrane, either that of the initial import vesicle or of some other vesicle formed from it (e.g. a lysosome or late endosome). If that's true, then aside from the argument about whether the inside of an uptake vesicle is topologically equivalent to the outside of the cell housing the vesicle, there's still the requirement for an additional molecular function to make the contained entities accessible to the rest of intracellular space.

Bottom line - perhaps until a *case of the sort I can't think of arises, the family relationship between uptake and transport remains OK.

Reported by: deustp01

Original Ticket: [geneontology/ontology-requests/10997](https://sourceforge.net/p/geneontology/ontology-requests/10997)

**Comment by dosumis** [comment]
The pattern is clear and easy to apply, but doing this completely would mean a relatively large number of new terms.  This is therefore a candidate for bulk addition using design pattern docs.

**Comment by pgaudet** [comment]
Endocytosis isn't transmembrane transport. The recent and ongoing changes to transport and transporter activity should make that clear.
